elements/bulbs-liveblog/bulbs-liveblog-entry.js

```diff
@@ -16,6 +16,10 @@ class BulbsLiveblogEntry extends BulbsHTMLElement {
     if (this.liveblog && this.liveblog.handleEntryAttached) {
       this.liveblog.handleEntryAttached({ target: this });
     }
+
+    if (twttr) {
+      twttr.widgets.load();
+    }
```

Comment:
Yeah, we're not ready for it yet, but that is the dream of a `<bulbs-content>` or `<div is="bulbs-content">` wrapper.

It would be an element that does any content/widget initialization in the `attachedCallback`.